docs/pages/CodeAnalysis.md

```diff
@@ -0,0 +1,227 @@
+# Code Analysis
+
+During the build process, UI5 Tooling executes a static code analysis of a project. In this and following sections the term "Simple Literial" is used. "Simple Literial" means following language fetaures:
+
+- String Literal
+- Numeric Literal
+- Boolean Literal
+- `null` / `undefined`
+- Template Literal
+- Arrays of the previous, spread operators will be ignored (not taken into account)
+- Object Literals with simple literals as keys and values (again, spread operators will be ignored / not taken into account)
+
+## Dependency Analysis
+
+UI5 Tooling extracts dependency information from a project's code as outlined in the following chapters.
+The APIs described in the following sections requiring the usage of "Simple Literals" when declaring dependencies.
+
+### JSModule Analyzer
+
+The [JSModule Analyzer](https://github.com/SAP/ui5-builder/blob/main/lib/lbt/analyzer/JSModuleAnalyzer.js) uses the ["Abstract Syntax Tree"](https://en.wikipedia.org/wiki/Abstract_syntax_tree) (AST) of a JavaScript file to decide whether a code block is executed *conditionally* or *unconditionally*.
```

Comment:
I would swap the order of the paragraphs (3, 1, 2 instead of 1,2,3). It is more important to know that certain APIs are detected to find dependencies. Only then one can IMO understand that the found dependencies are classified as standard, conditional or dynamic.

Reply:
Done